api/bash/create-teams.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Replace the "xxxxx" with the required values
+# Author: @ppremk
+
+# Script to create GitHub Teams in bulk on GitHub.com Organization
+# PAT Tokens needs to have the correct scope to be able to create teams in an organization
+# Teams are added as an Array. Teams are created as stand alone teams. Team relationship is not defined
+
+# To run the script:
+#
+# - Update VARS section in script
+# - chmod +x script.sh
+# - ./script.sh
+
+# VARS
+ORGNAME="xxxx" 
+PATTOKEN="xxxx"
+TEAMS=("team-name-1" "team-name-2")
+
+echo "Bulk creating teams in:"
+echo $ORGNAME
+
+for i in "${TEAMS[@]}"
+do
+curl --request POST \
```

Comment:
In order to make curl return an error code I think you need to pass the argument `--fail`